web/modules/custom/dept_topics/src/Commands/DeptTopicsCommands.php

```diff
@@ -0,0 +1,351 @@
+<?php
+
+namespace Drupal\dept_topics\Commands;
+
+use Drupal\Core\Database\Connection;
+use Drupal\Core\Database\Database;
+use Drupal\Core\Entity\EntityTypeManagerInterface;
+use Drupal\Core\StringTranslation\StringTranslationTrait;
+use Drupal\dept_migrate\MigrateSupport;
+use Drupal\dept_migrate\MigrateUuidLookupManager;
+use Drush\Commands\DrushCommands;
+
+/**
+ * Drush commands for dept_topics module.
+ */
+class DeptTopicsCommands extends DrushCommands {
+
+  use StringTranslationTrait;
+
+  /**
+   * The entity type manager service object.
+   *
+   * @var \Drupal\Core\Entity\EntityTypeManagerInterface
+   */
+  protected EntityTypeManagerInterface $etManager;
+
+  /**
+   * The migration lookup manager service object.
+   *
+   * @var \Drupal\dept_migrate\MigrateUuidLookupManager
+   */
+  protected MigrateUuidLookupManager $lookupManager;
+
+  /**
+   * The migration support service object.
+   *
+   * @var \Drupal\dept_migrate\MigrateSupport
+   */
+  protected MigrateSupport $migrateSupport;
+
+  /**
+   * D7 database connection.
+   *
+   * @var \Drupal\Core\Database\Connection
+   */
+  protected Connection $d7DbConn;
+
+  /**
+   * Class constructor.
+   */
+  public function __construct(EntityTypeManagerInterface $et_manager, MigrateUuidLookupManager $lookup_manager, MigrateSupport $migrate_support) {
+    parent::__construct();
+    $this->etManager = $et_manager;
+    $this->lookupManager = $lookup_manager;
+    $this->migrateSupport = $migrate_support;
+  }
+
+  /**
+   * Bulk generate any required topic and subtopic entity queues
+   * based on known stored values in the D7 database.
+   *
+   * @command dept_topics:syncTopicSubtopicOrder
+   * @aliases dept_topics:ststo
+   */
+  public function generateTopicAndSubtopicQueues() {
+    $this->io()->writeln("Syncing topics");
+    $this->syncTopics();
+
+    $this->io()->writeln("Syncing topics");
+    $this->syncSubtopics();
+
+    $this->io()->success("Done!");
+  }
+
+  /**
+   * Barebones function to return a dept id based on D7 domain id.
+   *
+   * @param int $d7_id
+   *   The D7 domain id.
+   *
+   * @return string
+   *   The D9 dept id.
+   */
+  private function domainD7ToD9(int $d7_id) {
```

Comment:
Couldn't find something open/service related that did the same thing as this so popped it in here (for now) - should be easy to extract into a service (new or existing) if it's useful in multiple contexts